rest-service/manager_rest/rest/resources_v2.py

```diff
@@ -761,6 +757,18 @@ def get(self, _include=None, filters=None,
         :rtype: :class:`manager_rest.storage.storage_manager.ListResult`
 
         """
+        if 'execution_id' not in filters:
```

Comment:
I'm not sure about this validation as it would probably be a good idea to allow filtering according to deployment_id, blueprint_id etc.. but lets assume this is fine for now.